src/main/java/Duke.java

```diff
@@ -1,10 +1,86 @@
+package src.main.java;
+import java.nio.file.attribute.UserDefinedFileAttributeView;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    private static int todoSize=0;
+    public static void printSeparator() {
+        System.out.println("----------------------------------------------");
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
+                    + "|  _ \\ _   _| | _____ \n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
+
         System.out.println("Hello from\n" + logo);
+        printSeparator();
+        System.out.println("Hello! I'm Duke\nWhat can I do for you?");
+        printSeparator();
+        String userInput;
+        ArrayList<Task> toDoList = new ArrayList<>();
```

Comment:
Considering renaming toDoList to list, since the list contains todos, deadlines and events.